app/views/dashboard/callouts/show.html.haml

```diff
@@ -0,0 +1,36 @@
+.col-12
+  .btn-toolbar{ role: 'toolbar' }
+    .btn-group.mr-2{ role: 'group' }
+      = button_to 'Back', :back, class: 'btn btn-outline-dark', method: :get
+    .btn-group.mr-2{ role: 'group' }
+      = button_to 'Edit', edit_dashboard_callout_path(@callout), class: ' btn btn-outline-primary', method: :get
+    .btn-group.mr-2{ role: 'group' }
+      = button_to 'Delete', dashboard_callout_path(@callout), class: ' btn btn-outline-danger', method: :delete, data: { confirm: 'Are you sure?' }
+    - if @callout.initialized?
+      .btn-group.mr-2{ role: 'group' }
+        = button_to 'Start', start_dashboard_callout_path(@callout), class: ' btn btn-outline-warning', method: :put, data: { confirm: 'Are you sure?' }
+    - else
+      - if @callout.paused? || @callout.stopped?
+        .btn-group.mr-2{ role: 'group' }
+          = button_to 'Resume', resume_dashboard_callout_path(@callout), class: ' btn btn-outline-warning', method: :put, data: { confirm: 'Are you sure?' }
+      - if @callout.running?
+        .btn-group.mr-2{ role: 'group' }
+          = button_to 'Stop', stop_dashboard_callout_path(@callout), class: ' btn btn-outline-warning', method: :put, data: { confirm: 'Are you sure?' }
+  %br
+.col-12.col-md-10
+  .card
+    .card-header
+      Callout detail
+    .card-body
+      %dl.row
+        %dt.col-sm-5.text-md-right ID:
+        %dd.col-sm-7= @callout.id
+      %dl.row
+        %dt.col-sm-5.text-md-right Status:
+        %dd.col-sm-7= @callout.status
+      %dl.row
+        %dt.col-sm-5.text-md-right Meta data:
+        %dd.col-sm-7= infinit_list(@callout.metadata)
+      %dl.row
+        %dt.col-sm-5.text-md-right Created at:
+        %dd.col-sm-7= @callout.created_at
```

Comment:
I feel it doesn't layout nicely

<img width="1246" alt="localhost_3000_dashboard_callouts_1" src="https://user-images.githubusercontent.com/131172/38175343-67546578-3605-11e8-9347-9ea2d9d39191.png">

Reply:
@samnang, if you have any ideas to help improve that.